src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -580,8 +718,17 @@ final class LazyList[+A] private(private[this] var lazyState: () => LazyList.Sta
     if (scout.isEmpty) state
     else tail.takeRightState(scout.tail)
 
+  /** @inheritdoc
+    *
+    * The first element returned by this element has the laziness semantics of [[take]],
+    * and the second element has the laziness semantics of [[drop]].
```

Comment:
It is not clear to me what you mean by “first” and “second” elements. I would just say that it preserves laziness for all except the first `from` elements.

Reply:
I completely confused this with `splitAt`, oops.